modules/cnf-about-nic-support-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_nic_support_irq_affinity_setting_{context}"]
+= About NIC support of IRQ affinity setting
```

Comment:
Probably the title is a bit missleading: the section's content says very little about NICs. May I suggest changing it to "About support of IRQ affinity setting"